website/docs/r/mdb_postgresql_cluster.html.markdown

```diff
@@ -194,6 +195,8 @@ The `user` block supports:
 
 * `permission` - (Optional) Set of permissions granted to the user. The structure is documented below.
 
+* `conn_limit` - (Optional) The maximum number of connections per user. (Default 50)
+
```

Comment:
The same.

Reply:
Name is from official docs https://cloud.yandex.com/docs/managed-postgresql/operations/cluster-users